plugins/commands/create.js

```diff
@@ -22,58 +84,89 @@ module.exports = () => ({
                 security: [{ token: [] }]
             }
         },
-        handler: (request, reply) => validator(request.payload.yaml)
-            .then((config) => {
-                if (config.errors.length > 0) {
-                    throw boom.badRequest(
-                        `Command has invalid format: ${config.errors.length} error(s).`,
-                        config.errors);
+        payload: {
+            parse: true,
+            allow: ['multipart/form-data', 'application/json']
```

Comment:
I'd like you to set `payload.maxBytes` here. I think the default 1MB setting is too small.
https://hapijs.com/api/3.0.0#-routeoptionspayloadmaxbytes

Reply:
I added `payload.maxBytes` to 1GB.